site/src/docs/components/header/index.mdx

```diff
@@ -344,6 +350,7 @@ The Header.NavigationMenu is enables users to navigate to different pages of the
       titleHref="https://hel.fi"
       logoAriaLabel="Service logo"
       logoHref="https://hel.fi"
+      logo={<Logo src={logoFi} />}
```

Comment:
The logo isn't showing in Colour variations sections. Maybe forgot to add the prop there?

Reply:
Ah... true. But but, now it's impossible to do it because they are images, not inline svg's.

Reply:
Oh this is different, sorry :D